src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -758,6 +768,21 @@ enum BikesAllowed {
     NOT_ALLOWED
 }
 
+input BikePreferences {
+    reluctance: Float
+    bikeWalking: WalkPreferences
+    speed: Float
+    switchTime: Int
+    switchCost: Int
+    optimize: OptimizationCriteria
+    boardCost: Int
```

Comment:
I would make Scalars for all common types we use: Speed, Duration, Cost, Slack and Reluctance

Reply:
Another question here, we probably still want to have costs that only support a static cost, for example `walkBoardCost`. Then we have use cases where we want to support a cost function (or alternatively a static cost). Should we have two scalar types Cost and CostFunction? Is cost always a positive integer? Does it make sense to have input wrapper type like I have used in the schema currently (Cost should be used instead of Float):
```
input Cost @oneOf {
    static: Float
    function: CostFunction
}
```